docroot/themes/humsci/humsci_basic/src/scss/components/_node--news-item.scss

```diff
@@ -63,18 +71,20 @@
 .views-field-field-hs-news-link a, // horizontal view link to external article default class
 .field-hs-news-link a {
   @include hb-colorful {
-    $hb-color--primary: hb-colorful-variation(primary);
     position: relative;
     margin-right: hb-calculate-rems(24px);
     display: inline-block;
-    border-bottom: 1px solid lighten(hb-colorful-variation(primary), 10%);
-
-    @include hb-link--hover-style(lighten(hb-colorful-variation(primary), 70%), (hb-colorful-variation(primary)));
+    border-bottom-width: hb-calculate-rems(1px);
```

Comment:
you can use the new border var here ` $hb-thin-border`

Reply:
hmmm actually should that be added to the `hb-link--hover-style` mixin? the border-width?

Reply:
That will be part of the STN-294. The reason we keep having issues with anchor tags is because of all the inconsistencies throughout the SCSS.